spec/policies.yml

```diff
@@ -0,0 +1,224 @@
+
```

Comment:
should this newline be here?

also, I think we would just call this file `policy.yml` and update the tag for this section to be just `policy`. if you look at all of the endpoints, I think this is more consistent

Reply:
but I guess all the others are plural - so if we change this we should change all of them. I'd look at the routes and see what's most consistent.